examples/md-flexible/measurePerf.sh

```diff
@@ -63,27 +63,30 @@ do
             # iterate over molecules with the correct repetition
             for i in `seq 0 $(( ${#Mols[@]} - 1 ))` ;
             do
+                thisReps=${Reps[$i]}
+                if [[ ${container} = 'DirectSum' && ${Mols[$i]} -ge 2048 ]];
```

Comment:
comment why you do this

Reply:
done cd1fdb996cce61c3a8cb54fa8e7afb3fa5fd9afb